packages/experimental/Expander/windows/ReactNativeExpander/ExpanderViewManager.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "winrt/Microsoft.ReactNative.h"
+#include "NativeModules.h"
+
+namespace winrt::ReactNativeExpander::implementation {
+
+    class ExpanderViewManager : public winrt::implements<
```

Comment:
nit: Typically, nested namespaces are used also in order to avoid extra indentation - you could shift everything to the left here